tasks/request_from_archive/test/unit_tests/test_request_from_archive.py

```diff
@@ -1857,23 +1857,12 @@ def test_restore_object_200_returned_raises(self, mock_logger_info: MagicMock):
     @patch("request_from_archive.task")
     def test_handler_happy_path(self, mock_task: MagicMock):
```

Comment:
All `test_handler`... tests should `mock set_optional_event_property`

Reply:
good catch. Added mock for `set_optional_event_property`